### PR TITLE
OSSM-5725: Add content to exclude the creation of CNI pods (2.5 Release)

### PR DESCRIPTION
**Pull request by rh-tokeefe**
**PR for Service Mesh 2.5 layered product release.**

Version(s): OCP 4.12+

Issue: https://issues.redhat.com/browse/OSSM-5725

Link to docs preview: https://70539--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-vs-community#ossm-cni_ossm-vs-istio

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]

**Comment by mkralik3** [comment]
LGTM

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

File: modules/ossm-vs-istio.adoc [hunk 1]
```diff
@@ -118,6 +118,12 @@ You can deploy virtual machines to OpenShift using OpenShift Virtualization. The
 
 {SMProductName} includes CNI plugin, which provides you with an alternate way to configure application pod networking. The CNI plugin replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to security context constraints (SCCs) with elevated privileges.
 
+[NOTE]
+====
+By default, Istio Container Network Interface (CNI) pods are created on all {product-title} nodes. To exclude the creation of CNI pods in a specific node, apply the  `maistra.io/exclude-cni=true` label to the node.
```
Comment (jewertow) [inline modules/ossm-vs-istio.adoc:123]:
```suggestion
By default, Istio CNI pods are created on all {product-title} nodes. To exclude the creation of CNI pods in a specific node, apply the  `maistra.io/exclude-cni=true` label to the node.
```

**Review by jewertow — CHANGES_REQUESTED** [review]

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by rh-tokeefe** [comment]
Rebased PR on March 18. PR was previously through merged review. Product code went live today. Please merge.

**Pull request merged**